### ISSUE TITLE
[DB] init configmap's name does not match the name in Values.yaml

### ISSUE THREAD

**Issue by jeanpommier** [georchestra/helm-georchestra#48]
The [configmap's name](https://github.com/georchestra/helm-georchestra/blob/main/templates/database/database-configmap-init.yaml#L5) is defined like this:
  `name: {{ include "georchestra.fullname" . }}-database-init`
When georchestra.fullname, defined in[ _helpers.tpl](https://github.com/georchestra/helm-georchestra/blob/main/templates/_helpers.tpl#L13) will most of the time be of the form `.Release.Name-$name`

While in the [Values.yaml](https://github.com/georchestra/helm-georchestra/blob/main/values.yaml#L220), it is called like this:
`          name: "{{ .Release.Name }}-database-init"`

So, there is a mismatch and the database won't start, because it can't find the configmap

**Comment by jeanpommier** [comment]
I don't think we can get the database subchart to use the georchestra's chart name. So hardcoding it in the values file might be a solution. What do you think of replacing by 
          `name: "{{ .Release.Name }}-database-init"`
in the values.yaml file ?

**Comment by edevosc2c** [comment]
What about changing in https://github.com/georchestra/helm-georchestra/blob/main/templates/database/database-configmap-init.yaml

From:
```
name: {{ include "georchestra.fullname" . }}-database-init
```

To:
```
name: {{ .Release.Name }}-database-init
```

**Comment by jeanpommier** [comment]
I'd say it doesn't exactly follow the naming logic, but fine for me

**Comment by edevosc2c** [comment]
Technical note and answer to this question:

> I don't think we can get the database subchart to use the georchestra's chart name.

This is because the bitnami postgresql chart is not aware of the helpers of the georchestra helm chart. It means this can't be done like this because it won't work:

```
extraVolumes:
      - name: "00-initsql"
        configMap:
          name: "{{ include "georchestra.fullname" . }}-database-init"
```

**Comment by jeanpommier** [comment]
Well, an alternative that, for me, would be quite acceptable (and was my first proposal) would be to simply hardcode the chart's name in the [dbinit config, in values.yaml](https://github.com/georchestra/helm-georchestra/blob/main/values.yaml#L244):
```
...
    extraVolumes:
      - name: "00-initsql"
        configMap:
          name: "{{ .Release.Name }}-georchestra-database-init"
...
```
This is not perfect, but very simple to adjust when changing the default name of the chart
